### PR TITLE
Comments infinite scroll

### LINKED ISSUE #2454: Infinite Scrolling - Lazy Load
Mechanics of loading additional comments is infinite scroll [Figma>](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2282%3A163649)

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #2454

---

You can test it on `/video/2`

### CODE REVIEW

File: packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx [hunk 1]
```diff
@@ -110,8 +119,32 @@ export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, vide
     return () => clearTimeout(timeout)
   })
 
+  // increase number of comments when user scrolls to the end of page
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx:102]: Unfortunately, this won't work well on smaller viewports, the comment section isn't last section on mobile: 
image
We would have to scroll down to CTA buttons and then scroll up to see new comments.

**Review by drillprop — CHANGES_REQUESTED** [review]
This could be tricky to do, but you didn't handle infinite scrolling for replies. There could be a comment that has 200 replies, we don't want to fetch them all at once.

**Comment by rafalpawlow** [comment]
@drillprop replies fetching will be resolved [here](https://github.com/Joystream/atlas/issues/2757)

**Review by drillprop — CHANGES_REQUESTED** [review]
We had a short call about current issues. Here's what we found:
- When you're on the video page and you switch users, the app breaks
- if you switch to another video and then switch back to the previous video, the number of comments is not correct, and infinite scrolling not working. This is probably cache problem
- The space is awkwardly small below the comments section on mobile
- Sometimes placeholders are blinking(they appear for a split second and disappear and then appear again) when you scroll fast - needs reproduction, I'm not sure how to trigger that. I'm also fine if we ignore this problem or fix it as a followup

**Review by drillprop — APPROVED** [review]

**Pull request merged**